app/renderer/components/Slider/Slider.tsx

```diff
@@ -0,0 +1,63 @@
+import * as React from "react";
+import "./Slider.scss";
+
+type SliderProps = {
+  max: string;
+  value: string;
+  children?: React.ReactNode;
+};
+
+type SliderStates = {
```

Comment:
I bet it should be `SliderState` instead of `SliderStates`, because component can have only one state which consists of various attributes